### DISCUSSION TITLE
react native support

### DISCUSSION THREAD

**Discussion by SMAsadAli** [react-dropzone/react-dropzone#1205] [Q&A]
Hi,
I am trying to use this package in my react-native project but getting `invariant violation: Text strings must be rendered within text strings` this error consistently.
SO is this package have support for react native or no, and if it does can you kindly share some examples on how to do a simple file picker.

**Comment by rolandjitsu** [comment]
@SMAsadAli I've never tested it, so it's not officially supported. If you managed to make it work, I would appreciate it if you share any insight on how you did it.
